prost-reflect-tests/src/json.rs

```diff
@@ -884,6 +883,19 @@ fn serialize_any_wkt() {
     );
 }
 
+#[test]
+fn serialize_options() {
+    let json = to_json(
+    Aggregate::default().descriptor().descriptor_proto()
```

Comment:
`descriptor_proto` returns a `prost_types::MessageDescriptor`, which doesn't have support for extension fields, so unfortunately there's no way for this to work until https://github.com/tokio-rs/prost/issues/1031 is implemented.

Reply:
I'm not sure why this is bound to the edition stuff.  custom options have been available in protobuf much before edition has been introduced. the nearest issue I see is tokio-rs/prost#754

Reply:
you're right, that is not the issue I intended to link. https://github.com/tokio-rs/prost/issues/674 tracks support for extension fields